src/app/loginComponents/accounts/account-sidebar/account-sidebar.component.scss

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2022 OICR and UCSC
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+@use '@angular/material' as mat;
+
+@import 'materialColorScheme.scss';
+
+div.p-3 {
+  width: 18vw;
+  min-width: 28rem;
+  height: 100%;
+  min-height: 100rem;
+  border-right: 0.1rem solid mat.get-color-from-palette($kim-gray, 4);
+  box-shadow: 0.1rem 0.2rem 0.1rem rgba(0, 0, 0, 0.1);
+}
+
+.bubble {
+  background-color: mat.get-color-from-palette($kim-accent1-success, 5);
+}
+
+img[alt='Dockstore avatar'] {
+  width: 18rem;
+  border-radius: 50%;
+  box-shadow: 0rem 0.2rem 0.4rem 0rem rgba(0, 0, 0, 0.1), 0rem 0.2rem 0.3rem 0rem rgba(0, 0, 0, 0.05);
+}
+
+// Synced badge
+.bubble.mt-2 {
+  border-radius: 1rem !important;
+  color: white !important;
+  background-color: $accent-3-dark;
+}
+
+img[alt='GitHub avatar'],
+img[alt='Google avatar'] {
+  width: 4.8rem;
+  border-radius: 50%;
+}
+
+table {
+  border: none;
+
+  th {
+    font-weight: 500;
+    color: $header-color;
+    padding: 0rem 1.5rem 0rem 0rem;
+  }
+
+  td {
+    color: mat.get-color-from-palette($kim-gray, 1);
+    padding: 0rem;
+  }
+}
```

Comment:
There are global styles for the new re-designed tables which cover a lot of these styles you defined: https://github.com/dockstore/dockstore-ui2/blob/e229c2dcdf4e1f61fa2e3e98997bb1ef60dba160/src/styles.scss#L1172

Downside is you need to convert your `<table>` to `<mat-table>`, but it would reduce style duplication.

The global one is missing `color: mat.get-color-from-palette($kim-gray, 1)` for the data cells so you could add a global class for `mat-cell` with this color. Poking around zeplin, it looks like other tables (like the search table) also use this color for the data cells

Reply:
mm so I don't think a `mat-table` will work. I remember trying it before but now that I'm retrying I remember why it may not be a good fit.

It doesn't look like it's possible to create a horizontal `mat-table`: [reference](https://stackoverflow.com/questions/59235658/how-to-create-a-horizontally-oriented-table-with-angular-material-mat-table)
The angular website also says it can "display rows of data" and didn't mention columns of data.  Looking further you can only define `matHeaderRowDef` but not a `matHeaderColumnDef`

Here's a quick sample with `mat-table` on top: (vertical table)
![image](https://user-images.githubusercontent.com/97123241/163856787-c6e5dd94-ece9-4812-ac20-825d758b615d.png)

Because of this I'd say it's best if we leave it as a regular html table... what I can do instead is add the html table tags to the `mat-table` styles:
* I can add just the `size-small` table so it doesn't interfere with any larger tables (I did a search and couldn't find any regular tables but just incase)

![image](https://user-images.githubusercontent.com/97123241/163860931-289cc337-e0ac-4f4f-bfd9-b39d36110dc1.png)
![image](https://user-images.githubusercontent.com/97123241/163861545-997f01c1-35d9-4502-812e-efb8bddb239a.png)

Adding the cell colour from Zeplin and changing the `font-weight` to `500` (roboto-medium) also from Zeplin. 

With above changes:
![image](https://user-images.githubusercontent.com/97123241/163861771-fe69961f-b70e-42d7-b243-d984f35c0509.png)

If you think this is a good alternative to the `mat-table` then I'll update this pr with the changes @kathy-t

Reply:
Yeah I think it's a good alternative. Did a quick search and you're right, it looks like all of our tables are mat-tables, so I don't think you need to make a `size-small` table.

When adding it to the global styles, double check if you need the padding ones for `table`. Material tables have their own custom padding (or lack thereof) so those padding styles were added.